### ISSUE TITLE
Get Out From Under Amazon’s Thumb

### ISSUE THREAD

**Issue by brethvoice** [getZION/self-hosted-box#3]
### Expected Behavior
I can keep my node even if Amazon goes out of business or tries to shut ZION (or me) down.

### Actual Behavior
I am forced to purchase from AWS who can cut me off at any time, or the entire ZION network.

### Your Environment
Uncertain, not relevant.

**Comment by masterial** [comment]
Amazon won't go out of business. It's would be pretty expensive to identify you with this setup.

**Comment by brethvoice** [comment]
@masterial please give a bit more detail.  Even if it is "expensive" for them to identify someone (I assume they would need to get their own ZION node, which they can easily handle) all they have to do is claim the entire ZION network violates their terms of service based on a single post with which they disagree...then it all goes down overnight.

**Comment by masterial** [comment]
They would have to blanket ban every single account that is hosting like this. I highly doubt this can happen overnight. 

Also, all the data that goes through it is encrypted with your private key that's on your phone. Even if they do ban it you'll still have your money and potentially database if you do proper backups. 

Ultimately, this is a kit for development and offers a way to get on Zion without paying subscription fees via our official channel. It just requires some manual handling.

**Comment by mryellow** [comment]
Self hosted? On AWS? That's an oxymoron.

Unless Zion can simply be ran, it won't find an true adoption.

**Comment by darcysabatino** [comment]
I simply won’t run a service in AWS because.. I just don’t want to. It’s a dockerized service, so I don’t see why I would require an AWS instance. I should be able to run the docker service in my own system.

An easy self-host option could be a Raspberry Pi image that users can deploy on their own network. Take a look at the Umbrel project. Even better idea - work with the Umbrel project team to have a one-click Zion install on a personal Umbrel node! That would be very sweet, and would reduce the barrier, while encouraging folks to run their own Bitcoin and Lightning node, adding even more security to the network.

**Comment by edc1934** [comment]
Any roadblocks to just running the scripts on a local server?

**Comment by masterial** [comment]
@edc1934 You need a way to be able to connect from the outside of the network. 🤷🏼‍♂️ We don't know how to do it.

**Comment by 0evmc** [comment]
> 

There are lots of ways - VPN; reverse proxy with certificates; if you’re brave: port forwarding (should put node in a DMZ). 

people have joined nodes to BitCoin network from home for a while - reference that as a potential starting point.

**Comment by masterial** [comment]
@0evmc It's not that simple cowboy. Need a way to able to issues requests from mobile. Basically, a static IP/domain thing. Please understand the problem before suggesting solutions. ☮️

**Comment by masterial** [comment]
@0evmc Touche! At this point, it's a priority thing.

**Comment by edc1934** [comment]
> 

Makes sense but more complicated than what I'm willing to do. Would be nice if I could just run in a container / VM at home and have some easy VPN option direct from the container.

**Comment by edc1934** [comment]
@0evmc Makes sense but more complicated than what I'm willing to do. Would be nice if I could just run in a container / VM at home and have some easy VPN option direct from the container.

**Comment by 0evmc** [comment]
@edc1934, Yeah, hosting anything from home can be a lot to deal with, especially if you don't particularly enjoy all the nuts and bolts of the networking piece. 
They do make services to take all that hassle off your plate though - AWS comes to mind...😜😬

**Comment by mryellow** [comment]
> @0evmc It's not that simple cowboy. Need a way to able to issues requests from mobile. Basically, a static IP/domain thing. Please understand the problem before suggesting solutions. peace_symbol

A VPN. TailScale works simply on mobile.

**Comment by masterial** [comment]
@mryellow which configs?

**Comment by mryellow** [comment]
I haven't delves too far into it, The name "Zion" and the fringe content turned me off.

Box setup and docker environment all talks about AWS as if it's required.

https://github.com/getZION/self-hosted-box#initial-setup
